bin/varnishtest/tests/r03560.vtc

```diff
@@ -0,0 +1,29 @@
+varnishtest "A backend connection error gets returned as a valid short response"
+
+barrier b1 sock 2
+
+server s1 {
+	rxreq
+	txresp -nolen -hdr "Content-Length: 10"
+	barrier b1 sync
+	send "12345"
+	# Early connection close error
+} -start
+
+varnish v1 -vcl+backend {
+	import vtc;
+
+	sub vcl_deliver {
+		# Make sure we are streaming and give the backend time to error out
+		vtc.barrier_sync("${b1_sock}");
+		vtc.sleep(1s);
+	}
+} -start
+
+client c1 {
+	txreq
+	rxresphdrs
+	expect resp.http.Content-Length == "10"
+	recv 5
```

Comment:
`rxrespbody` and then check that what's been received is indeed "12345".

Reply:
I think its more important to show the `expect_close` at the end. Also, it seems like `rxrespbody` doesn't work on a response like this.

Reply:
RTFM?

> Receive (part of) a response's body.

Reply:
Sorry, just to be crystal clear, `rxrespbody` will always result in a VTC failure since the body is incomplete. Maybe I’m missing something?

Reply:
I agree with the VTC as is